pwiz_tools/Skyline/CommandArgs.cs

```diff
@@ -902,6 +902,10 @@ private bool ParseReintegrateExcludeFeature(NameValuePair pair)
             (c, p) => c.Refinement.QValueCutoff = p.ValueDouble);
         public static readonly Argument ARG_REFINE_MINIMUM_DETECTIONS = new RefineArgument(@"refine-minimum-detections", INT_VALUE,
             (c, p) => c.Refinement.MinimumDetections = p.ValueInt);
+        public static readonly Argument ARG_REFINE_SC_INCLUDED_CUTOFF = new RefineArgument(@"refine-shape-correlation-included-cutoff", NUM_VALUE,
+            (c, p) => c.Refinement.SCIncludedCutoff = p.ValueDouble);
+        public static readonly Argument ARG_REFINE_SC_QUANTIZATION_CUTOFF = new RefineArgument(@"refine-shape-correlation-quantization-cutoff", NUM_VALUE,
```

Comment:
Thanks for the feedback! I changed the CommandArgs based on Brendan's feedback below. But I changed variables in RefineDlg and RefinmentSettings to use Quantitative instead of Quantization